components/selectors/parser.rs

```diff
@@ -1299,16 +1306,18 @@ fn parse_qualified_name<'i, 't, P, E, Impl>
 }
 
 
-fn parse_attribute_selector<'i, 't, P, E, Impl>(parser: &P, input: &mut CssParser<'i, 't>)
-                                                -> Result<Component<Impl>,
-                                                          ParseError<'i, SelectorParseError<'i, E>>>
-    where P: Parser<'i, Impl=Impl, Error=E>, Impl: SelectorImpl
+fn parse_attribute_selector<'i, 't, P, Impl>(parser: &P, input: &mut CssParser<'i, 't>)
```

Comment:
Feel free to reformat the signatures to use block indentation for arguments if you feel like it while touching them.